tests/test_gsobject_model.py

```diff
@@ -0,0 +1,483 @@
+    if noise != 0:
+        gn = galsim.GaussianNoise(sigma=noise, rng=rng)
+        star.image.addNoise(gn)
+    return star
+
+
+def test_simple():
+    """Initial simple test of Gaussian, Kolmogorov, and Moffat PSFs.
+    """
+    # Here is the true PSF
+    size = 1.3
+    g1 = 0.23
+    g2 = -0.17
+    cenu = 0.1
+    cenv = 0.4
+    for fiducial in [fiducial_gaussian, fiducial_kolmogorov, fiducial_moffat]:
+        psf = fiducial.dilate(size).shear(g1=g1, g2=g2).shift(cenu, cenv)
+
+        # Draw the PSF onto an image.  Let's go ahead and give it a non-trivial WCS.
+        wcs = galsim.JacobianWCS(0.26, 0.05, -0.08, -0.29)
+        image = galsim.Image(64,64, wcs=wcs)
+
+        # This is only going to come out right if we (unphysically) don't convolve by the pixel.
+        psf.drawImage(image, method='no_pixel')
+
+        # Make a StarData instance for this image
+        stardata = piff.StarData(image, image.trueCenter())
+        star = piff.Star(stardata, None)
+
+        # First try fastfit.
+        print('Fast fit')
+        model = piff.GSObjectModel(fiducial, fastfit=True)
+        fit = model.fit(star).fit
+
+        print('True size = ',size,', model size = ',fit.params[0])
+        print('True g1 = ',g1,', model g1 = ',fit.params[1])
+        print('True g2 = ',g2,', model g2 = ',fit.params[2])
+        print('True cenu = ', cenu, ', model cenu = ', fit.center[0])
+        print('True cenv = ', cenv, ', model cenv = ', fit.center[1])
+
+        # This test is fairly accurate, since we didn't add any noise and didn't convolve by
+        # the pixel, so the image is very accurately a sheared Kolmogorov.  Only wrinkle is that
+        # HSM is adapted to Gaussians, not Kolmogorovs, so the requirements aren't as strict as
+        # for Gaussian.
+        np.testing.assert_allclose(fit.params[0], size, rtol=5e-2)
+        np.testing.assert_allclose(fit.params[1], g1, atol=1e-4)
+        np.testing.assert_allclose(fit.params[2], g2, atol=1e-4)
+        np.testing.assert_allclose(fit.center[0], cenu, atol=1e-3)
+        np.testing.assert_allclose(fit.center[1], cenv, atol=1e-3)
+
+        # Now try fastfit=False.
+        print('Slow fit')
+        model = piff.GSObjectModel(fiducial, fastfit=False)
+        fit = model.fit(star).fit
+
+        print('True size = ',size,', model size = ',fit.params[0])
+        print('True g1 = ',g1,', model g1 = ',fit.params[1])
+        print('True g2 = ',g2,', model g2 = ',fit.params[2])
+        print('True cenu = ', cenu, ', model cenu = ', fit.center[0])
+        print('True cenv = ', cenv, ', model cenv = ', fit.center[1])
+
+        # The sensitivity of this test is somewhat dependent on what the wcs is, and whether the fit
+        # center parameters are in image coords or world coords.
+        np.testing.assert_allclose(fit.params[0], size, rtol=1e-3)
+        np.testing.assert_allclose(fit.params[1], g1, atol=1e-3)
+        np.testing.assert_allclose(fit.params[2], g2, atol=1e-3)
```

Comment:
These tolerances for g1,g2 are less strict than with fastfit.  Was this intentional?  Or can we tighten them up?